src/osd/OSD.cc

```diff
@@ -2512,6 +2513,25 @@ void OSD::create_logger()
 
   PerfCountersBuilder osd_plb(cct, "osd", l_osd_first, l_osd_last);
 
+  // Latency axis configuration for op histograms, values are in nanoseconds
+  PerfHistogramCommon::axis_config_d op_hist_x_axis_config{
+    "Latency (usec)",
+    PerfHistogramCommon::SCALE_LOG2, ///< Latency in logarithmic scale
+    0,                               ///< Start at 0
+    100000,                          ///< Quantization unit is 100usec
```

Comment:
@liewegas To clarify the quantization stuff: I'm finding bucket in two steps, first is to quantize and shift the input space to normalize it. In the next step the quantized value is converted to a final bucket number by moving to a correct scale - either leave it as linear or make logarithmic, this second transformation always operate on uniform input space then.

Regarding the output - for each bucket it also dumps the range of values it holds so I guess switching to something else would just require dumping it in compatible format. Dumping probabilistic data structure could be a bit trickier though as long as we don't want to output large amounts of data.